spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb

```diff
@@ -166,6 +166,32 @@
         parser.instance_variable_get('@data_index')[:container_image][:by_ref_and_registry_host_port].values[0])
       expect(parser.instance_variable_get('@data')[:container_images][0][:architecture]).to eq('amd64')
     end
+
+    context "image registries from openshift images" do
+      after :each do
+        inventory = {"image" => [image_from_openshift,]}
+
+        parser.get_openshift_images(inventory)
+        expect(parser.instance_variable_get('@data_index')[:container_image_registry][:by_host_and_port].size).to eq(1)
+        expect(parser.instance_variable_get('@data')[:container_image_registries].size).to eq(1)
```

Comment:
Doing expectations in an `after` is very strange, as `after` is meant for cleanup/teardown, and it especially makes the `it` on line 179 very strange as it's empty.  If you want to bunch these together to DRY it up, prefer a regular Ruby method that is called from each `it`, or perhaps even better, use RSpec `shared_examples`